src/renderer/components/ChatView.js

```diff
@@ -118,8 +119,8 @@ class ChatView extends React.Component {
 
   render () {
     const { attachmentMessage, setupMessage } = this.state
-    const chat = this.getChat()
-    if (!chat) return (<div />)
+    const chat = this.props.chat
```

Comment:
I think this is where we broke updates. It's using a cached value.

Reply:
`chat` should be the chat object from the `deltachat` json and it has to be retrieved each time _or_ the calling component needs to update this prop.